demo/admin/src/common/apollo/createApolloClient.ts

```diff
@@ -6,6 +6,7 @@ import fragmentTypes from "@src/fragmentTypes.json";
 export const createApolloClient = (apiUrl: string) => {
     const httpLink = new HttpLink({
         uri: `${apiUrl}/graphql`,
+        credentials: "include",
```

Comment:
Is this always necessary or just locally?

To my understanding, we already use cookies for authentication. But since all API requests are routed through the oauth proxy, admin and API have the same domain. Meaning, the cookies are included in the requests out-of-the-box (since [`same-origin` is the default](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#supplying_request_options:~:text=same%2Dorigin%20(the%20default)))

So this should work out-of-the-box in deployed envs and `credentials: "include"` should only be necessary locally. Or not?

Reply:
So this won't be a breaking change in our projects?